#Goal
float64 x
float64 y
---
#Result
bool success
float64 final_x
float64 final_y
---
#Feedback
float64 current_x
float64 current_y
float64 distance_to_goal